tests/integration/test_postgresql_database_engine/test.py

```diff
@@ -321,17 +333,20 @@ def test_predefined_connection_configuration(started_cluster):
 
 
 def test_postgres_database_old_syntax(started_cluster):
-    conn = get_postgres_conn(started_cluster, True)
+    conn = get_postgres_conn(
+        started_cluster.postgres_ip, started_cluster.postgres_port, database=True
+    )
     cursor = conn.cursor()
 
     node1.query(
         """
-        DROP DATABASE IF EXISTS test_database;
-        CREATE DATABASE test_database ENGINE = PostgreSQL('postgres1:5432', 'test_database', 'postgres', 'mysecretpassword', 1);
+        CREATE DATABASE postgres_database ENGINE = PostgreSQL('postgres1:5432', 'postgres_database', 'postgres', 'mysecretpassword', 1);
         """
     )
     create_postgres_table(cursor, "test_table")
-    assert "test_table" in node1.query("SHOW TABLES FROM test_database")
+    assert "test_table" in node1.query("SHOW TABLES FROM postgres_database")
+    cursor.execute(f"DROP TABLE test_table ")
+    node1.query("DROP DATABASE IF EXISTS postgres_database;")
```

Comment:
I suppose we have to do it in
```
try:
    ....
finally:
    cursor.execute(f"DROP TABLE test_table ")
    node1.query("DROP DATABASE IF EXISTS postgres_database;")
```

Reply:
Why? We should not fail in the test. And it is easier to understand what happens without complex logic. What if we failed because server is not working and we can't drop as well. We will see second error and first one can be lost etc.